core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2797,6 +2797,15 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_JOURNAL_SYNC_ROOT_AFTER_INIT_FROM_BACKUP =
+      booleanBuilder(Name.MASTER_JOURNAL_SYNC_ROOT_AFTER_INIT_FROM_BACKUP)
+          .setDefaultValue(true)
+          .setDescription("Controls whether to automatically invalidate the root of the UFS after"
```

Comment:
maybe just change from the language from invalidate to needs synchronization to be consistent